server/lib/nl/constants.py

```diff
@@ -155,10 +155,75 @@
 
 # TODO: remove this special casing when a better NER model is identified which
 # can always detect these.
-SPECIAL_PLACES: Set[str] = {'palo alto', 'mountain view'}
+SPECIAL_PLACES: Set[str] = {
```

Comment:
Given your context from the chat is it worth calling this OVERRIDE_FOR_NER?

Reply:
done